java/sha_digest.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+VERSION="3.0.0"
+
+
+FILE_NAME=sha-digest-version.txt
+CURRENT_DATE_TIME=$(date '+%Y-%m-%dT%T.%zZ')
+SHA_256_VALUE=$(cd anaplan-connect-client/target; openssl dgst -sha256 anaplan-connect-client-${VERSION}-jar-with-dependencies.jar | cut -d=  -f2)
```

Comment:
We need to add this in release steps that once the branch is merged to master, run a mvn clean install / package and create a sha of that jar.

Reply:
done